Sources/PackageDescription/PackageDescription.swift

```diff
@@ -23,11 +23,12 @@ import Foundation
 /// statement to provide the name of the package, its targets, products,
 /// dependencies, and other configuration options.
 ///
-/// By convention, the properties of a `Package` are defined in a single nested
```

Comment:
If we drop the code voice, should "package" be lower case?

Reply:
True!